pdq/LCOM.go

```diff
@@ -0,0 +1,14 @@
+package pdq
+
+import "math"
+
+func CalcLCOM(nonSharedAttributePairs int, sharedAttributePairs int) int {
+	absoluteNonSharedPairs := math.Abs(float64(nonSharedAttributePairs))
+	absoluteSharedPair := math.Abs(float64(sharedAttributePairs))
```

Comment:
I'm strictly following the formula where the variables are wrapped in a module. 
Although, in theory, negative values will never be passed to the function.
This makes that both options have rights to exist. Do you still want me to remove conversions?
![lcom](https://github.com/4rchr4y/go-code-metric/assets/71561797/8bdbbb2c-232e-45e5-b672-a2405501361b)

Reply:
okay, so here I can see several problems 
1. In other functions we a doing the negative cheks. Therefore, looks like an unexpected behavior of the library function compared to other functions
2. btw, why the name of func atrs is `...AttributePairs` it theory we are working with a struct/class fields, so prbalpy could be `...Fields`
3. Why do u nedd `math.Abs` function? In my understanding we can get rid of `math.Abs` calls since absolute value is not needed

What do u think about this code?

```golang
func CalcLCOM(nonSharedAttributePairs int, sharedAttributePairs int) int {

  if nonSharedAttributePairs < 0 || sharedAttributePairs < 0 {
    return -1 // or another error code
  }

  if nonSharedAttributePairs >= sharedAttributePairs {
    return nonSharedAttributePairs - sharedAttributePairs 
  }

  return 0
}
```

The key changes:

- remove unnecessary conversions to float64
- remove `math.Abs` calls since absolute value is not needed
- added check for negative inputs and return error
- directly compare `nonSharedAttributePairs` vs `sharedAttributePairs` as ints

Will it work for u?